### ISSUE TITLE
High loss value with alpha = 0.5 and alpha 1.

### ISSUE THREAD

**Issue by Socret360** [Oreobird/tf-keras-deep-head-pose#5]
I am having an issue with high loss value. I was able to start the training process and notice that the loss value is very high (3000+ with alpha = 1 ) and (1000+ with alpha = 0.5) as opposed to 10+ mentioned in the paper. Initially, i thought the high loss values only happens in the first few epochs. However, that is not the case. The loss value is still high (at least compared to the 15+ mentioned in the paper).

I suspect that there could be something wrong with the input or maybe the loss calculation. I would really appreciate it if anyone could help me with this.

**Comment by Socret360** [comment]
It turns out that I have incorrectly filtered the filenames from the dataset leading to a mismatch between the input and label.
